### PR TITLE
Kusto: generate package-2020-02-15

### PR DESCRIPTION
**Pull request by xseeseesee**

### CODE REVIEW

File: sdk/kusto/pom.mgmt.xml [hunk 1]
```diff
@@ -13,5 +13,6 @@
     <module>mgmt-v2019_05_15</module>
     <module>mgmt-v2019_09_07</module>
     <module>mgmt-v2019_11_09</module>
+    <module>mgmt-v2020_02_15</module>
```
Comment (ChenTanyi) [inline sdk/kusto/pom.mgmt.xml:16]: I think this module could be added automatically. Is it due to you didn't add it after generation?

Comment (weidongxu-microsoft) [reply sdk/kusto/pom.mgmt.xml:16]: But version and relativePath in pom.xml is correct. So I think Chuang is using the latest?

Comment (xccc-msft) [reply sdk/kusto/pom.mgmt.xml:16]: Last PR didn't get merged due CI failure at that time. I just closed the pending one and use this new instead.

File: sdk/kusto/mgmt-v2020_02_15/src/main/java/com/microsoft/azure/management/kusto/v2020_02_15/implementation/ClustersInner.java [hunk 2]
```diff
@@ -1581,8 +1581,16 @@ public DiagnoseVirtualNetworkResultInner call(ServiceResponse<DiagnoseVirtualNet
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
      * @return the List&lt;ClusterInner&gt; object if successful.
      */
-    public List<ClusterInner> list() {
-        return listWithServiceResponseAsync().toBlocking().single().body();
+    public PagedList<ClusterInner> list() {
```
Comment (ChenTanyi) [inline sdk/kusto/mgmt-v2020_02_15/src/main/java/com/microsoft/azure/management/kusto/v2020_02_15/implementation/ClustersInner.java:1584]: Is it any reason to change this return type?

Comment (xccc-msft) [reply sdk/kusto/mgmt-v2020_02_15/src/main/java/com/microsoft/azure/management/kusto/v2020_02_15/implementation/ClustersInner.java:1584]: The conflict comes from `InnerSupportsListing`, which `ClustersInner` implements.

**Pull request merged**